recipes-devtools/ruby-cross_2.7.2.bb

```diff
@@ -0,0 +1,117 @@
+# Modified version from poky upstream + adjustments from https://github.com/voxtok/meta-oad-ruby/tree/dizzy/recipes-devtools/ruby
+# SPDX-License-Identifier: MIT, BSD-2-Clause
+# Copyright (c) 2015, Michaël Burtin
+# Copyright (c) 2020, Konrad Weihmann
+# Copyright (c) 2020, Yocto maintainers
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/ruby-cross:"
+
+SUMMARY = "An interpreter of object-oriented scripting language"
+DESCRIPTION = "Ruby cross variant"
+HOMEPAGE = "http://www.ruby-lang.org/"
+SECTION = "devel/ruby"
+LICENSE = "Ruby | BSD | GPLv2"
+LIC_FILES_CHKSUM = "\
+    file://COPYING;md5=5b8c87559868796979806100db3f3805 \
+    file://BSDL;md5=19aaf65c88a40b508d17ae4be539c4b5\
+    file://GPL;md5=b234ee4d69f5fce4486a80fdaf4a4263\
+    file://LEGAL;md5=2b6d62dc0d608f34d510ca3f428110ec \
+"
+
+DEPENDS += "\
+            gdbm \
+            libffi \
+            libyaml \
+            openssl \
+            readline \
+            ruby-native \
+            zlib \
+           "
+
+PN = "ruby-cross-${TARGET_ARCH}"
+
+# Ignore how TARGET_ARCH is computed.
+TARGET_ARCH[vardepvalue] = "${TARGET_ARCH}"
+
+SHRT_VER = "${@oe.utils.trim_version("${PV}", 2)}"
+
+SRC_URI_append = " \
```

Comment:
potential issue: Patch '0001-Makefile-cross-compile-fixes.patch' should contain an Upstream-Status entry